input/fsh/profiles/PractitionerAttendant.fsh

```diff
@@ -0,0 +1,48 @@
+Profile: PractitionerBirthAttendant
+Parent: PractitionerVitalRecords
+Id: Practitioner-birth-attendant
+Title: "Practitioner - Birth Attendant"
+Description: "The attendant at birth is defined as the individual at the delivery who is responsible for the delivery."
+* name.text ^short = "Attendant's name"
+* identifier ^short = "Note: required by USCore"
+* identifier[NPI] ^short = "Attendant's NPI Number"
+* qualification 0..1
+* qualification 
+  * ^slicing.discriminator.type = #value
+  * ^slicing.discriminator.path = "code"
+  * ^slicing.rules = #open
+  * ^slicing.description = "Slicing based on the profile conformance of the sliced element"
+* qualification contains birthAttendantQualification 0..1
```

Comment:
Is the qualification needed in both the Attendant and Certifier profiles?

Reply:
Yes, I believe so. That was an oversight when we pulled it from PractitionerVR, but I will add it to Certifier